Documentation/etcd-mixin/mixin.libsonnet

```diff
@@ -219,6 +219,32 @@
               message: 'etcd instance {{ $labels.instance }} HTTP requests to {{ $labels.method }} are slow.',
             },
           },
+         {
+            alert: 'etcdDatabaseSizeAlmostFull',
+            expr: |||
+              (etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100 > 95
+            ||| % $._config,
+            'for': '10m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'etcd cluster "{{ $labels.job }}": database size is almost full on etcd instance {{ $labels.instance }}, please defrag or increase the quota as the writes to etcd will be disabled when it is full.',
+            },
+         },
+         {
+            alert: 'etcdDatabaseWriteSurge',
+            expr: |||
+              increase(((etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100)[240m:1m]) > 50
```

Comment:
nit `etcdExcessiveDatabaseGrowth`?

`increase()` can show results in unexpected ways based on the sample period. Should we write a test[1] for this query? I can help with that.

[1] https://github.com/etcd-io/etcd/blob/45ac89a18bb254c001d7bda46341ed8b9ac95c74/Documentation/etcd-mixin/test.yaml

Reply:
Writing a test for the query makes sense @hexfusion. We have a test case in place as part of the latest commit, will look into testing it as per the instructions in the Readme in the repo and will let you know if there are any questions. Thanks.